src/python_inspector/resolution.py

```diff
@@ -224,71 +289,64 @@ def get_requirements_for_package_from_pypi_simple(self, candidate):
             repos=self.repos,
             python_version=python_version,
         )
-        for wheel in wheels:
-            deps = list(
-                PypiWheelHandler.parse(os.path.join(utils_pypi.CACHE_THIRDPARTY_DIR, wheel))
-            )
-            assert len(deps) == 1
-            deps = deps[0].dependencies
-            for dep in deps:
-                if dep.scope == "install":
-                    yield packaging.requirements.Requirement(str(dep.extracted_requirement))
-
-        sdist_file = fetch_and_extract_sdist(
-            repos=self.repos, candidate=candidate, python_version=python_version
-        )
 
-        if sdist_file:
-            setup_py_path = os.path.join(
-                sdist_file,
-                "setup.py",
-            )
-            setup_cfg_path = os.path.join(
-                sdist_file,
-                "setup.cfg",
+        has_wheels = False
+
+        for wheel in wheels:
+            wheel_path = os.path.join(utils_pypi.CACHE_THIRDPARTY_DIR, wheel)
```

Comment:
since this is an absolute location, I would rather call this `wheel_path`
```suggestion
            wheel_location = os.path.join(utils_pypi.CACHE_THIRDPARTY_DIR, wheel)
```